Casks/d/drawpile.rb

```diff
@@ -3,21 +3,43 @@
     version "2.1.7"
     sha256 "820ec2837d3c7ea4e190a64cbb1d0fdecb3797bf968277b3b7ca0cc5d758987f"
 
+    url "https://github.com/drawpile/Drawpile/releases/download/#{version}/Drawpile.#{version}.dmg",
+        verified: "github.com/drawpile/Drawpile"
+
     livecheck do
       skip "Legacy version"
     end
   end
-  on_sierra :or_newer do
+  on_sierra do
     version "2.1.20"
     sha256 "fe7f93c2f3ec9505b8a4f044093b67bae0c80fc8d6613ba2d0dfad5243cfdf44"
 
+    url "https://github.com/drawpile/Drawpile/releases/download/#{version}/Drawpile.#{version}.dmg",
+        verified: "github.com/drawpile/Drawpile"
+
     livecheck do
-      url "https://drawpile.net/files/osx/"
-      regex(/href=.*?Drawpile[ ._-]v?(\d+(?:\.\d+)+)\.dmg/i)
+      skip "Legacy version"
+    end
+  end
+  on_high_sierra :or_newer do
+    version "2.2.1"
+    sha256 "d4b29c78da9a64eb8a5526c464f9647b48a99b60cace0ce3eaf06a4e484dec60"
+
+    url "https://github.com/drawpile/Drawpile/releases/download/#{version}/Drawpile-#{version}.dmg",
+        verified: "github.com/drawpile/Drawpile"
+
+    livecheck do
+      url :url
+      regex(/^(\d+(?:\.\d+)+)$/)
+      strategy :github_latest do |json, regex|
+        match = json["tag_name"]&.match(regex)
+        next if match.blank?
+
+        (match[1]).to_s
+      end
```

Comment:
```suggestion
  strategy :github_latest
```

The block isn't required as the default strategy should suffice.